requirements.txt

```diff
@@ -17,6 +18,7 @@ progressbar
 pyDNS
 pylibravatar
 pyramid
+pyramid_debugtoolbar
```

Comment:
This is not a production requirement, and in fact you wouldn't want it present on production. If we add this to this file, then the production package won't install without this dependency so we should drop this line.

Reply:
Thanks for the feedback! I moved it to developer docs.